sql/moz-fx-data-shared-prod/firefox_accounts_derived/health_lighthouse_v1/metadata.yaml

```
@@ -0,0 +1,12 @@
+friendly_name: Accounts Health Lighthouse
+description: >
+  Front-end health metrics for Firefox Accounts, calculated from Lighthouse reports.
+owners:
+  - wclouser@mozilla.com
+external_data:
+  format: google_sheets
+  source_uris:
+    - https://docs.google.com/spreadsheets/d/1uGHOS-cY_OUqicbBkS2Z20EK0hLq3PDw28TZcxP5RHI/edit?gid=1783447085 # URL to the spreadsheet
+  options:
```

> When I add a spreadsheet as a source directly in bigquery I can specify what tab to use.  I didn't see that in [the instructions](https://docs.telemetry.mozilla.org/cookbooks/operational/connecting_external_data_bigquery.html#connecting-sheets) and I'm not sure where to add it to the yaml.  The data is on the "Lighthouse" tab in my spreadsheet.

I believe this should do that:
```suggestion
  options:
    range: Lighthouse
```

---

thanks